src/openforms/authentication/signals.py

```diff
@@ -142,4 +143,5 @@ def set_cosign_data_on_submission(
     form_auth = request.session.get(FORM_AUTH_SESSION_KEY)
 
     instance.co_sign_data = form_auth
+    instance.co_sign_data["cosign_date"] = timezone.localtime().isoformat()
```

Comment:
why not `timezone.now().isoformat()`? Django's datetimefields store their values in the DB as UTC too and having that consistency is nice.